README.md

```diff
@@ -19,44 +19,45 @@ As of 06/11/2017, this project is in the "working prototype" phase of developmen
 
 1. [Raspberry Pi](https://www.raspberrypi.org/products/)
 1. Raspberry Pi [camera module](https://www.raspberrypi.org/products/camera-module-v2/)
-1. SD card with NOOBS (minimum 4GB)
-1. Network access to the Pi (wifi for Pi3, ethernet for Pi2)
+1. SD card with Raspbian GNU/Linux 10 OS (minimum OS and SD Card size 4GB)
+1. Network access to the Pi (WiFi for Pi3, WiFi/ethernet for Pi2)
 1. Suggest wall mountable case for the Pi ([Vilros](http://www.vilros.com/) has a good case with a mount for the camera)
+1. Tactile button and female-to-female jumper lead to get input (By default, missed-moment uses GPIO 26/Pin 37 and the Ground on Pin 39 to attach the button [GPI Reference](https://www.raspberrypi.org/documentation/usage/gpio/)) 
```

Comment:
Lets also link to this options:
https://www.amazon.com/Warmstor-3-Pack-Desktop-Button-Computer/dp/B072FMVZJZ/ref=sr_1_3?dchild=1&keywords=2+pin+pc+desktop+power+cable&qid=1596030325&sr=8-3